spec/minimax_server/router_spec.clj

```diff
@@ -0,0 +1,37 @@
+(ns minimax-server.router-spec
+  (:import [scarvill.httpserver.response Response]
+           [scarvill.httpserver.request Request RequestBuilder Method])
+  (:require [speclj.core :refer :all]
+            [minimax-server.router :refer :all]
+            [clojure.string :as str]))
+
+(describe "router"
+
+  (defn request-with [current-player board]
+    (-> (new RequestBuilder)
+        (.setParameter "current_player" current-player)
+        (.setParameter "board" board)
+        (.setURI "/")
+        .build))
+
+  (defn get-body-as-str [response]
+    (String. (.getBody response)))
+
+  (def example-request (request-with "x" "x,o,x,o,x,o,x,o,_"))
+
+  (it "returns a response"
```

Comment:
While this test was probably useful for driving out behavior, subsequent tests make it unnecessary. It can probably be deleted.